content/docs/work-with-source-git.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Source-git"
+date: 2020-06-25T23:59:59+01:00
+draft: false
+chapter: false
+weight: 2
+aliases:
+- /source-git
+- /omfg
+---
+
+This document is focused on how packit treats a source-git repo.
+
+In short, source-git concept is fairly simple - take an upstream repo and shove
+downstream content on top of it. Easy right?
+
+Packit is then able to work with such a repo and:
+
+* Create SRPMs
+
+* Sync the state into dist-git
+
+* And you are able to utilize all features of packit-service: builds on PRs,
+  tests on those builds, update your package in Fedora.
+
+Let's describe all of these in detail.
+
+
+### Upstream history or not
```

Comment:
This should be improved, but atm I have no clue how.

Reply:
let's see what others on the team think

Reply:
just the title or the section?

Reply:
I hope the whole section

Reply:
Actually I was referring to the title only (that's why the comment was placed on this line, instead of being attached to all the lines in the section stuck_out_tongue): I find it unclear. By reading it even I have difficulties gaining a quick understanding what this section will be about, and I can imagine readers unfamiliar with the concept of source-git will be even more lost.